blocks/form/custom-components/range/_range.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
having a separate folder for custom-components makes it more complex IMO.
we should use the same folder and this would make it easier to load the custom component without the hard coded name in mappings.js (I'm not in favor of custom component hard coded names in mappings.js - this makes it an extra unintuitive step wrt sites to write a custom component)

Reply:
I agree with you on the hardcoding of `mappings.js` but I think a custom component folder is a necessary step towards separation of concerns, especially with larger projects that have many custom components, it helps keeping the OOTB decorators modularized.